### DISCUSSION TITLE
Email under Settings menu missing

### DISCUSSION THREAD

**Discussion by borisasadanin** [EyevinnOSC/community#4] [General]
I understand that the email should be displayed under the Settings tab. Mine is empty. 
For all the reported bugs, I´m using the Safari browser v17.1.2 on Mac.

<img width="1018" alt="image" src="https://github.com/Eyevinn/osaas/assets/66821532/1cd064a0-8965-4786-b0bb-514c23018624">

**Comment by bwallberg** [comment]
Thanks for your feedback! This has been fixed.
